COCO_DATA_FORMAT.md

```diff
@@ -227,4 +227,28 @@ people_dataset/
     train_coco_location.json
     test_coco_location.json
 ```
-```
\ No newline at end of file
+
+
+## KeyValuePair dataset
+
+KeyValuePair dataset is associated with a schema defining a task that input is text + (multi-)image, label is key-value pairs. Each sample of dataset consists of
```

Comment:
Might be more clear:

"It is a generic variation of an image-text dataset, where the input consists of one or more images and a text. The output is represented as a dictionary, where keys are the attributes of interests."